doc/whats_new/v0.21.rst

```diff
@@ -35,7 +35,10 @@ random sampling procedures.
   seed, including :class:`linear_model.LogisticRegression`,
   :class:`linear_model.LogisticRegressionCV`, :class:`linear_model.Ridge`,
   and :class:`linear_model.RidgeCV` with 'sag' solver. |Fix|
-
+- Any model using the :func:`svm.libsvm` or the :func:`svm.liblinear` solver,
```

Comment:
I don't think these are funcs. Just call them LibSVM and LibLinear

Reply:
They actually are functions: 

 - From `svm/base.py`: `from . import libsvm, liblinear`.
 - Also they are declared in `svm/__init__.py` file.

But if you think that's more clear, I can still call them LibSVM and LibLinear :)